src/shared/qrcode-util.c

```diff
@@ -89,6 +108,56 @@ static void write_qrcode(FILE *output, QRcode *qr) {
         fflush(output);
 }
 
+static void write_positioned_qrcode(FILE *output, QRcode *qr, unsigned int row, unsigned int column) {
+        assert(qr);
+
+        if (!output)
+                output = stdout;
+
+        int fd, move_down = 2;
```

Comment:
2. This value ```move_down = 2```, I also got after trying different initial values: 1, 3, 4. It was 2 that eliminated the blank line I had between the top border and the qr code, I need to understand that fully, so it gets standardized.

Reply:
add a comment above to explain why it is set as such